instructor.html

```diff
@@ -72,58 +73,292 @@ <h6 class="mdc-drawer__subtitle" id="userEmailDisplay"></h6>
-              </div>
-              <div class="card-body">
-                <div
-                  class="alert alert-danger"
-                  id="instructorErrorAlert"
-                  style="display: none"
-                ></div>
-                <table
-                  class="table table-striped"
-                  id="instructorsTable"
-                  style="min-width: 100%"
-                >
-                  <thead>
-                    <tr>
-                      <th>Instructor's Name</th>
-                      <th>Instructor's Bio</th>
-                      <th>Instructor's Role</th>
-                      <th>Instructor's TwitterId</th>
-                      <th>Instructor's GithubId</th>
-                      <th>Actions</th>
-                    </tr>
-                  </thead>
-                  <tbody></tbody>
-                </table>
-              </div>
-              <div class="card-footer text-right">
-                <button class="btn btn-primary" id="addButton" disabled>
-                  Add Instructor
-                </button>
-              </div>
-            </div>
-            <!-- Sticky Footer -->
-            <footer class="sticky-footer">
-              <div class="container my-auto">
-                <div class="copyright text-center my-auto">
-                  <span>Copyright © DSC FUTA 2018</span>
+          <!-- card-grid -->
+          <div class="card-grid">
+            <h1 class="grid-heading mdc-typography--headline3">
+                Instructors</h1>
+
+            <ul class="mdc-image-list my-image-list">
+                <li class="mdc-image-list__item">
+                    <div class="mdc-layout-grid__cell mdc-card mdc-card--outline">
+                        <div class="mdc-card__primary-action">
+                        <div class="mdc-card__media">
+                            <div class="user-icon-container">
+                                <span class="material-icons user-icon">
+                                    person_outline
+                                </span>
+                            </div>
+                        </div>
+                        <!-- ... additional primary action content ... -->
+                        <div class="card-content">
+                            <h3 class='card__name'>Instructor's name</h3>
+                            <p class='card__role'>Role</p>
+                            <small class="card__description">Short description of the instructor and what he/she does
+                            </small>
+                            <div class="handle">
+                                <h4>Twitter ID</h4>
+                                <small>Twitter ID</small>
+                            </div>
+                            <div class="handle">
+                                <h4>Github ID</h4>
+                                <small>Github ID</small>
+                            </div>
+                        </div>
```

Comment:
About this, kindly take a look at the following file, lines 231 to 238, for a reference on the pre-existing classes you can use for these:
https://github.com/dscfuta/vibranium-admin/blob/3bfb52c0d42e1cac88649db83070a9ab46c74881/projects.html#L231

Prefer using Material Web Component styling, where available, in order to maintain a consistent look across pages.

Reply:
done